src/components/ManageNotifications.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+
+import { deleteItem, getData, postData } from '@utils/apiHooks';
+import { formatUIDate } from '@utils/date';
+import { USER_VOYAGE_REPORT_URL, VOYAGE_REPORT_URL } from '@constants/ApiConstants';
+import { EDIT_VOYAGE_CHECK_DETAILS_URL } from '@constants/ClientConstants';
+
+const ManageNotifications = (pageData) => {
+  const [tabData, setTabData] = useState([]);
+  const [tableName, setTableName] = useState('Draft');
+  const [reportList, setReportList] = useState();
+  const tabs = [
+    {
+      name: 'draft',
+      text: 'Draft',
+      active: true,
+    },
+    {
+      name: 'submitted',
+      text: 'Submitted',
+      active: false,
+    }, {
+      name: 'cancelled',
+      text: 'Cancelled',
+      active: false,
+    },
+  ];
+
+
+  const setActiveTab = (e) => {
+    const tabArray = [...tabData];
+    tabArray.map((elem) => {
+      if (e.target.id === elem.name) {
+        elem.active = true;
+        setTableName(e.target.id.charAt(0).toUpperCase() + e.target.id.slice(1));
+      } else {
+        elem.active = false;
+      }
+    });
+    setTabData(tabArray);
+  };
+
+
+  const getReportList = () => {
+    let validReports = [];
+    getData(USER_VOYAGE_REPORT_URL)
+      .then((resp) => {
+        if (!resp.errors) {
+          resp.items.map((report) => {
+            if (!report.departureDate && !report.departureTime && !report.departurePort) {
+              deleteItem(`${VOYAGE_REPORT_URL}/${report.id}`);
+            } else {
+              validReports.push(report);
+            }
+          });
+          setReportList(validReports);
+        }
+      });
+  };
+
+  useEffect(() => {
+    setTabData(tabs);
+    getReportList();
+  }, [pageData]);
+
+  if (!pageData || !tabData || tabData.length === 0) { return null; }
+  return (
+    <div className="govuk-width-container">
+      <main className="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+        <h1 className="govuk-heading-l">Manage your notifications</h1>
+        <p className="govuk-body-l">Click on one of your notifications below to view, edit, cancel or delete it, depending on its status. For example, you can edit a draft or submitted notification, but you can&apos;t edit a cancelled notification.</p>
+        <div className="govuk-tabs" data-module="govuk-tabs">
+          <ul className="govuk-tabs__list">
+            {tabData && tabData.map((tab) => {
+              return (
+                <li
+                  key={tab.name}
+                  className={tab.active === true ? 'govuk-tabs__list-item govuk-tabs__list-item--selected' : 'govuk-tabs__list-item'}
+                  onClick={(e) => setActiveTab(e)}
+                >
+                  <p id={tab.name} className="govuk-tabs__tab">
+                    {tab.text}
+                  </p>
+                </li>
+              );
+            })}
```

Comment:
No need to use `return` here:
```suggestion
            {tabData && tabData.map((tab) => (
                <li
                  key={tab.name}
                  className={tab.active === true ? 'govuk-tabs__list-item govuk-tabs__list-item--selected' : 'govuk-tabs__list-item'}
                  onClick={(e) => setActiveTab(e)}
                >
                  <p id={tab.name} className="govuk-tabs__tab">
                    {tab.text}
                  </p>
                </li>
              );
            )}
```

Reply:
I think you do need it because it's using map method? otherwise it won't return the elements for each tab